Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -62,7 +62,6 @@ internal final class CommentsViewController: UITableViewController {
 
     self.commentComposer.delegate = self
 
-    self.tableView.dataSource = self.dataSource
```

Comment:
I don't think this line needs to be removed? Putting it back in did not break anything from my observation.